components/clearbutton/index.css

```diff
@@ -65,13 +65,25 @@
 		--mod-clear-button-icon-color-down: var(--spectrum-disabled-content-color);
```

Comment:
Are we keeping these `--mod-clear-button-icon-color-hover` and `--mod-clear-button-icon-color-down` after all?

Reply:
Ah, yep! I'm leaving those in place as they are referenced elsewhere. ✨